beluga_tutorial/src/main.cpp

```diff
@@ -224,15 +222,14 @@ int run(const std::filesystem::path& path) {
     record.current = particles;
 
     particles |= beluga::actions::propagate(std::execution::seq, motion_model);
-    record.propagate = particles;
+    record.prediction = particles;
 
     particles |= beluga::actions::reweight(std::execution::seq, sensor_model) | beluga::actions::normalize;
-    record.reweight = particles;
 
     particles |= beluga::views::sample |                                        //
                  ranges::views::take_exactly(parameters.number_of_particles) |  //
                  beluga::actions::assign;
```

Comment:
nit: We can define the update stage like this:

```
particles |= beluga::actions::reweight(std::execution::seq, sensor_model) |  //
             beluga::actions::normalize |                                    //
             beluga::views::sample |                                         //
             ranges::views::take_exactly(parameters.number_of_particles) |   //
             beluga::actions::assign;
```

Reply:
Maybe in this code, we can write the update stage this way. However, in the explanation of the code, I still want to separate the stages for explanatory purposes.